hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaSSLProperties.java

```diff
@@ -0,0 +1,197 @@
+package pl.allegro.tech.hermes.management.config.kafka;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "ssl")
+public class KafkaSSLProperties {
```

Comment:
in fact it is a copy of the `pl.allegro.tech.hermes.common.config.KafkaSSLProperties` properties. Why we can't use that ?